testing/u-boot-s5p6818/APKBUILD

```diff
@@ -0,0 +1,36 @@
+# Contributor:
+# Maintainer:
+pkgname=u-boot-s5p6818
+pkgver=1.2
+pkgrel=0
+pkgdesc="u-boot for s5p6818"
+url="https://github.com/rafaello7/u-boot-nanopi-m3"
+arch="aarch64"
+license="BSD-2"
```

Comment:
BSD-2-Clause [spdx](https://spdx.org/licenses/BSD-2-Clause.html)

Reply:
@maxice8 Fixed! Thank you!